go/oasis-test-runner/oasis/runtime.go

```diff
@@ -52,6 +54,7 @@ type RuntimeCfg struct { // nolint: maligned
 	Keymanager  *Runtime
 	TEEHardware node.TEEHardware
 	MrSigner    *sgx.MrSigner
+	Version     string
```

Comment:
This could actually be `version.Version` and the parsing should happen in the fixture's `Create` function.